### PR TITLE
Add Nginx v1.8.1 template

### PR DESCRIPTION
**Pull request by rayandas**
Related: https://github.com/rancher/rancher/issues/41840

### CODE REVIEW

File: pkg/rke/templates/templates.go [hunk 1]
```diff
@@ -271,7 +272,8 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 			">=1.23.16-rancher1-1 <1.24.0-rancher1-1":  nginxIngressv151Rancher2,
 			">=1.24.2-rancher1-1 <1.24.10-rancher1-1":  nginxIngressv120Rancher2,
 			">=1.24.10-rancher1-1 <1.26.0-rancher1-1":  nginxIngressv151Rancher2,
-			">=1.26.0-rancher1-1":                      nginxIngressv170Rancher1,
+			">=1.26.0-rancher1-1 <1.27.2-rancher1-1":   nginxIngressv170Rancher1,
+			">=1.27.2-rancher1-1":                      nginxIngressv181Rancher1,
```
Comment (jiaqiluo) [inline pkg/rke/templates/templates.go]: Question: why Nginx v1.8.1 is used for v1.27.2 and above, not v1.27.0 and above?

**Review by jiaqiluo — CHANGES_REQUESTED** [review]
I have only one question, other than that, the PR LGTM. 


For reference, the only changes in the upstream template are the version bumps: https://github.com/kubernetes/ingress-nginx/compare/controller-v1.7.0...controller-v1.8.1

File: pkg/rke/templates/templates.go (hunk 1 above)
Comment (rayandas) [reply pkg/rke/templates/templates.go]: We were introducing 1.27.2 version back then for 1.27 support so I kept it as 1.27.2. will change it to 1.27.0 and update the PR.

Comment (rayandas) [reply pkg/rke/templates/templates.go]: Update the PR

**Review by jiaqiluo — APPROVED** [review]

**Comment by rayandas** [comment]
@kinarashah could you please approve this PR as well?

**Comment by rayandas** [comment]
I've tested the changes. It's working fine.

**Review by kinarashah — APPROVED** [review]

**Pull request merged**